src/components/FeaturePanel/PublicTransport/PublicTransportWrapper.tsx

```diff
@@ -12,3 +15,47 @@ export const PublicTransportWrapper = ({ children }) => {
 
   return <div style={divStyle}>{children}</div>;
 };
+
+const fmtCategory = (category: string) => {
+  switch (category) {
+    case 'tourism':
+      return t('publictransport.tourism');
+    case 'night':
+      return t('publictransport.night');
+    case 'car_shuttle':
+      return t('publictransport.car_shuttle');
+    case 'car':
+      return t('publictransport.car');
+    case 'commuter':
+      return t('publictransport.commuter');
+    case 'regional':
+      return t('publictransport.regional');
+    case 'long_distance':
+      return t('publictransport.long_distance');
+    case 'high_speed':
+      return t('publictransport.high_speed');
+    default:
+      return t('publictransport.unknown');
+  }
+};
+
+interface CategoryProps {
+  category: string;
+  lines: LineInformation[];
+  amountOfCategories: number;
+}
+
+export const PublicTransportCategory: React.FC<CategoryProps> = ({
+  category,
+  lines,
+  amountOfCategories,
+}) => (
+  <>
+    {amountOfCategories > 1 && <h4>{fmtCategory(category)}</h4>}
+    <PublicTransportWrapper>
+      {lines.map((line) => (
+        <LineNumber name={line.ref} color={line.colour} />
```

Comment:
One idea - could you please try to make this a link to that specific relation? It should be very easy and add some more value. I would expect the buttons to look all the same, just do some change on hover (opacity or underline)

Reply:
Turns out it is comp lexer then we thought! Ideally the link should be to a train route itself. But that isn't what's fetched and shown - we show one direction of a train route.  
Here is a example: On Frankfurt central station we probably want to link to [this](https://www.openstreetmap.org/relation/2445543) relation but instead we link to [this one](https://www.openstreetmap.org/relation/2078076). What do you say?

Reply:
I think it is ok to link to "one direction" of the route for now. It is still better than nothing. 

It can be partially solved by fetch parentFeatures for these "one directions" routes, which will at least allow the user to display the master.

Reply:
You probably saw it in the review but just for completeness I also write it here:
Now the masters and normal routes are fetched. For the geojson on the map the normal routes are used while the masters are used for the featurepanel.